scargo/file_generators/templates/conan/profile.j2

```diff
@@ -5,6 +5,9 @@ CXX=/opt/gcc-arm-none-eabi/bin/arm-none-eabi-g++
 {% elif config.project.target.family == "esp32" %}
 CC=$ENV{IDF_PATH}/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/xtensa-esp32-elf-gcc
```

Comment:
For esp32 CC and CXX doesn.t have any effect . It is dead code and can be safely removed. Compiler is set by toolchain file here

` tools.cmake.cmaketoolchain:toolchain_file=/opt/esp-idf/tools/cmake/toolchain-esp32.cmake`

Reply:
Created separate issue: https://github.com/Spyro-Soft/scargo/issues/287